types/plotly.js/tsconfig.json

```diff
@@ -18,6 +18,8 @@
     },
     "files": [
         "index.d.ts",
-        "plotly.js-tests.ts"
+        "lib/core.d.ts",
```

Comment:
Hi @sheetalkamat, thanks for taking a look! If you look at `lib/core.d.ts`, it is simply doing an `export * from 'plotly.js';`. The copied-and-pasted code is the test, which can be removed. The goal with this PR is to allow users to import from `plotly.js/lib/core` in addition to plain old `plotly.js` while still importing type declarations.